common/src/main/java/org/iris_events/common/message/IdentityAuthenticated.java

```diff
@@ -0,0 +1,10 @@
+package org.iris_events.common.message;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.iris_events.annotations.Message;
+
+import java.util.UUID;
+
+@Message(name = "identity/authenticated")
```

Comment:
`identity/authenticated` can be a constant defined within [Exchanges.java](https://github.com/iris-events/iris/blob/main/common/src/main/java/org/iris_events/common/Exchanges.java)